Documentation/Development/SolutionStructure.md

```diff
@@ -0,0 +1,19 @@
+# WheelMUD Solution Structure
+
+We have tried to organized the code into logical groupings, in solution folders and projects. The current solution structure looks like this:
+
+![SolutionExplorer](img/WMSolutionExplorer.gif)
+
+Here's a brief explanation of each solution folder: 
+
+| Folder          | Description |
+| --------------- | ----------- |
+| Administration  | This is where anything related to editing the server contents or configuration goes. |
```

Comment:
I still think we should allow people to edit things in game so that can be exported from a local instance and imported into the game.  I'm not a fan of having it have to be external through a separate tool; I think its important that builders be able to do so on a local instance, perhaps even using a backup of production to do so, and then have the ability to export that work, check it in, and then load it to production via import.